src/main/resources/de/qabel/desktop/ui/accounting/menuqr/menuqr.fxml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.layout.VBox?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.shape.Line?>
+
+<?import javafx.scene.layout.AnchorPane?>
+<AnchorPane fx:id="menuQR" xmlns="http://javafx.com/javafx/8" visible="false" styleClass="popup-menu"
+            xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.qabel.desktop.ui.accounting.menuqr.MenuQRController">
+    <children>
+        <VBox fx:id="vboxMenu" styleClass="vbox-menu">
+            <HBox>
+                <children>
+                    <Label styleClass="icon-edit">
+                        <HBox.margin>
+                            <Insets left="5" bottom="3" top="3"/>
+                        </HBox.margin>
+                    </Label>
+                    <Button fx:id="labelEdit" text="%editDetails" styleClass="button-menu"/>
```

Comment:
it's a button, not a label. Either call id `fx:id="editButton"` or just omit the detail and call it `fx:id="editDetails"`